boards/chronos/board_init.c

```diff
@@ -16,7 +16,7 @@
  * @brief       eZ430-chronos board initialization
  *
  * @author      Oliver Hahm <oliver.hahm@inria.fr>
- * @author      Ludwig Ortmann <ludwig.ortmann@fu-berlin.de>
+ * @author      Ludwig Knüpfer <ludwig.knuepfer@fu-berlin.de>
```

Comment:
Yes. When I exmatriculated I entered the [Ernst-Reuter-Gesellschaft der Freunde, Förderer und Ehemaligen der Freien Universität Berlin e.V.](http://www.fu-berlin.de/sites/alumni/erg/). After the name change I updated my name there and after a while this change fluctuated into the FU system. Once that happened I was able to add a new address for my new name (I am keeping my old address as well) in the regular zedat interface.
On a related note: I was able to get my certificate in my new name without any hassle as well. (I changed names before I presented the thesis).